charts/ark-cluster/templates/deployment.yaml

```diff
@@ -209,6 +209,8 @@ spec:
             - mountPath: /arkconfig
               name: {{ $chart_name }}-config-volume
             - mountPath: /ark/config
+              name: {{ $chart_name }}-config-bis-volume
```

Comment:
what does 'bis' stand for? should it be 'base'?

Reply:
As you want, it's actually confusing to have /ark/config and /arkconfig

You want me to name it base then ?

I will run test and push later. I was at the airport when I made this pr ahah.